### DISCUSSION TITLE
Log in problem

### DISCUSSION THREAD

**Discussion by 404foundOwO** [ppy/osu#27571] [Q&A] [answered]
This happened maybe after i changed my dns to download something. I did try network reset through Windows settings, osu!stable works fine and other apps too. Other in-game things like wiki,changelog,news etc. wont load. But somehow it still gets and downloads updates. 

![the issue](https://github.com/ppy/osu/assets/105450937/e9566924-060f-468b-9731-94c38f55686b)

Hope these are the right logs:
[1710180678.database.log](https://github.com/ppy/osu/files/14563601/1710180678.database.log)
[1710180678.auth.log](https://github.com/ppy/osu/files/14563602/1710180678.auth.log)
[1710180678.network.log](https://github.com/ppy/osu/files/14563603/1710180678.network.log)

**Comment by smoogipoo** [comment]
Please also provide the runtime.log (or just upload all logs).

**Comment by smoogipoo** [comment, marked as answer]
Googling for the issue suggests some sort of Windows-level corruption. I found this suggestion, that could help: https://stackoverflow.com/a/29370236 (`netsh winsock reset`).

But realistically at the point where you've made changes to your system to cause something like this, it's completely out of our hands.

**Reply by 404foundOwO** [reply]
It fixed the issue, thanks!
